workbench-core/repo-scripts/github-oidc/README.md

```diff
@@ -0,0 +1,142 @@
+# `github-oidc`
+
+⚠️ $\textcolor{red}{\text{Experimental}}$ ⚠️ : Not for use in any critical, production, or otherwise important deployments
+
+## Description
+This package creates the roles in an AWS account that can be assumed from a github workflow. The permissions given to these roles enable the workflow to perform deployments in the specified AWS account. 2 CDK stacks are created as part of this package:
+1. `OIDCProviderStack`: This stack deploys the identity provider for GitHub
+2. `<GitHub-Org>-GitHubOIDCStack`: This stack deploys the role-to-assume which will be used in our github workflows to deploy and perform integration/e2e tests
+
+## Constructs
+This package defines 2 constructs:
+1. [MafGithubOidcRolePolicy](./src/constructs/maf-github-oidc-role-policy.ts)
+2. [SwbGithubOidcRolePolicy](./src/constructs/swb-github-oidc-role-policy.ts)
+
+Use these constructs to update the permissions according to the requirement.
+
+### MAF Example Package Instructions
```

Comment:
If you're not going to use "workbench-core" consistent with the rest of this project, you at lest need to define "MAF" here before using the acronym.